docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,70 @@
+# Signalen and the Standard for Public Code
+
+## [Code in the open](https://standard.publiccode.net/criteria/code-in-the-open.html)
+
+* Current backlog and bug-tracking is not public. Some issues referenced in code and commits are not easy to locate, like "SIG-1525". Transition to public bug-tracker?
+* Can the classification system be completely open sourced? How can we address risks and possibilities regarding (of theoretical) leaking of information. Maybe add anonymized MOR datasets or redo the classification algorithm with public available MOR data sets. Can a basic Common Ground version be made?
+* Citizens are not authenticated. Data is not cleaned up for sharing (privacy purposes). https://github.com/Amsterdam/signals/blob/master/docs/api-design.md#publiek-en-ongeauthenticeerd
```

Comment:
We will remove this point, because:
- by design citizens can leave a complaint/notification in SIA without logging-in
- the second point is relevant to reporting/machine learning but out of scope for now

Reply:
removed in 8a706682d73c43c5caa5117817ef8b4d30775f1f